scripts/entry.js

```diff
@@ -219,6 +248,24 @@ function Entry(data,host)
     while(m.indexOf("{-") > -1 && m.indexOf("-}") > -1){
       m = m.replace('{-',"<del>").replace('-}',"</del>");
     }
+    var il;
+    var ir;
+    while((il = m.indexOf("{%")) > -1 && (ir = m.indexOf("%}")) > -1){
+      var left = m.substring(0, il);
+      var mid = m.substring(il + 2, ir);
+      var right = m.substring(ir + 2);
+
+      var origin = this.quote && this.target ? this.target : this.host.url;
+      var src = origin + '/media/content/inline/' + mid;
+
+      if (src.indexOf('.') == -1) {
+          src = src + '.png'; // Default extension: .png
```

Comment:
Wouldn't default `svg` be more rontonde-y?

Reply:
You're right. At the time, I picked `png` due to how most emotes I found were `png`, but didn't change that when merging -mod into -client.

@neauoire Do we want to change it? To keep old messages intact, we could check if the entry was created before a given timestamp, use `png` as default for those entries and `svg` for newer ones.

Reply:
I don't know, I think we should keep png. It's more likely that someone will share non svg content.